### PR TITLE
MORE CRIMES FOR JAILBIRD

### PR DESCRIPTION
**Pull request by WonderPsycho**
adds more crimes for the jailbird trait

# About The Pull Request
adds new crime strings/prompts for the random_crime.txt file

## Why It's Good For The Game

adds more random crime veriaty for jailbird trait and more hilarious security rp!

### CODE REVIEW

File: monkestation/strings/random_crimes.txt [hunk 1]
```diff
@@ -59,3 +59,16 @@ Reached Skate Nirvana, Flubbed Up
 Too Cool
 Cooking Reality-Bending Food
 Recycling
+Caused a Intergalactic War Between NanoTrasen And The Clown Union, Due To Adding Silentium To a Banana
+Stun Prodded a Clown And Spacing Said Clown Out a Airlock
```
Comment (nednaZ) [inline monkestation/strings/random_crimes.txt:63]: This one might need to go. They're meant to be obviously fake crimes when looked into.

File: monkestation/strings/random_crimes.txt [hunk 2]
```diff
@@ -59,3 +59,16 @@ Reached Skate Nirvana, Flubbed Up
...
+Blew Up a Station With a Supermatter Delamination
+Murderboned TOO HARD
+Drugged a Station's Crew With Floorpills
+Has a Engine Addiction that Causes More Harm Then Good
+Unleashed a Chaos Dunk On 15 Different Space Stations, They Were Stopped by a Security Officer On One Station Before They Did Another Chaos Dunk On Said Station, Many Know Them As "The Chaos Dunk Maniac"
+Was Caught Killing Other Crimers And Was Arrested For Vigilante Justice
+Was a Intern That Stole Very Confidential [REDACTED] NT Documents From Central Command, Was Swiftly Fired By Nanotrasen and Arrested By Nanotrasen Central Command Security
+Stole a Ancient Clown Banana Artifact From The Clown Union
+Bio-Terrorism
+Released a Mutegen Virus That Turned Many into Mutant Clowns And Flooded The Station With Plasma
```
Comment (nednaZ) [inline monkestation/strings/random_crimes.txt:73]: Spelling error on this one

File: monkestation/strings/random_crimes.txt [hunk 3]
```diff
@@ -59,3 +59,16 @@ Reached Skate Nirvana, Flubbed Up
 Too Cool
 Cooking Reality-Bending Food
 Recycling
+Caused a Intergalactic War Between NanoTrasen And The Clown Union, Due To Adding Silentium To a Banana
+Stun Prodded a Clown And Spacing Said Clown Out a Airlock
+Blew Up a Station With a Supermatter Delamination
+Murderboned TOO HARD
+Drugged a Station's Crew With Floorpills
+Has a Engine Addiction that Causes More Harm Then Good
+Unleashed a Chaos Dunk On 15 Different Space Stations, They Were Stopped by a Security Officer On One Station Before They Did Another Chaos Dunk On Said Station, Many Know Them As "The Chaos Dunk Maniac"
```
Comment (nednaZ) [inline monkestation/strings/random_crimes.txt:68]: Length of this one might be too much for the actual consoles

Comment (WonderPsycho) [reply monkestation/strings/random_crimes.txt:68]: maybe.... we have to text to see

File: monkestation/strings/random_crimes.txt (hunk 1 above)
Comment (WonderPsycho) [reply monkestation/strings/random_crimes.txt:63]: which one, the stun prodded clown one?

Comment (nednaZ) [reply monkestation/strings/random_crimes.txt:63]: Yes

Comment (WonderPsycho) [reply monkestation/strings/random_crimes.txt:63]: aight will get to removing that one

Comment (WonderPsycho) [reply monkestation/strings/random_crimes.txt]: removed the stun prodded clown one

File: monkestation/strings/random_crimes.txt (hunk 2 above)
Comment (WonderPsycho) [reply monkestation/strings/random_crimes.txt]: fixed the mutagen spelling error
and a another spelling error

**Review by nednaZ — APPROVED** [review]
Looks good to me.

**Comment by WonderPsycho** [comment]
woah i did not expect this to get merged already i had plans on testing it today later, guess you tested it before i got to, welp off to get ready on working on my next PR, which i'll talk about more on the discord

**Pull request merged**